python/pyspark/sql/tests/test_functions.py

```diff
@@ -84,9 +85,6 @@ def test_function_parity(self):
 
         # Functions that we expect to be missing in python until they are added to pyspark
         expected_missing_in_py = {
-            # TODO(SPARK-50220): listagg functions will soon be added and removed from this list
-            "listagg_distinct",
-            "listagg",
             "string_agg",
```

Comment:
do we plan to add the aliases `string_agg` and `string_agg_distinct`?

Reply:
Yes, ofc. Just wanted to achieve green ci for `listagg` first

Reply:
added